pom.xml

```diff
@@ -103,35 +109,6 @@
   </dependencies>
 
   <build>
-    <plugins>
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-enforcer-plugin</artifactId>
```

Comment:
I hope you enjoyed it, @orrc . It was a real PITA :(

Reply:
Yeah…